airflow/providers/mysql/CHANGELOG.rst

```diff
@@ -27,6 +23,22 @@ used with MySQL server 5.6.4 through 5.7.
 Changelog
 ---------
 
+5.0.0
+.....
+
+Breaking changes
+~~~~~~~~~~~~~~~~
+
+.. warning::
+  If you are using Remove mysql-connector-python please note that starting this release it is an extra option.
+  The library will not be installed by default.
```

Comment:
:+1: . BTW - looks like Oracle gets serious about upgrading the connector  after some poking :)

Reply:
Regardless leaving it as optional extra is a good thing